src/PrestaShopBundle/Resources/views/Admin/Component/Layout/search_form.html.twig

```diff
@@ -0,0 +1,66 @@
+{#**
+ * Copyright since 2007 PrestaShop SA and Contributors
+ * PrestaShop is an International Registered Trademark & Property of PrestaShop SA
+ *
+ * NOTICE OF LICENSE
+ *
+ * This source file is subject to the Open Software License (OSL 3.0)
+ * that is bundled with this package in the file LICENSE.md.
+ * It is also available through the world-wide-web at this URL:
+ * https://opensource.org/licenses/OSL-3.0
+ * If you did not receive a copy of the license and are unable to
+ * obtain it through the world-wide-web, please send an email
+ * to license@prestashop.com so we can send you a copy immediately.
+ *
+ * DISCLAIMER
+ *
+ * Do not edit or add to this file if you wish to upgrade PrestaShop to newer
+ * versions in the future. If you wish to customize PrestaShop for your
+ * needs please refer to https://devdocs.prestashop.com/ for more information.
+ *
+ * @author    PrestaShop SA and Contributors <contact@prestashop.com>
+ * @copyright Since 2007 PrestaShop SA and Contributors
+ * @license   https://opensource.org/licenses/OSL-3.0 Open Software License (OSL 3.0)
+ *#}
+<form id="header_search"
+      class="bo_search_form dropdown-form js-dropdown-form collapsed"
+      method="post"
+      action="{{ link.getAdminLink('AdminSearch') }}"
+      role="search">
+  <input type="hidden" name="bo_search_type" id="bo_search_type" class="js-search-type" />
+  {% if show_clear_btn is defined and show_clear_btn %}
```

Comment:
Don't you need to inject `show_clear_btn` as well then?

Reply:
you're right